ITEventsTests/FavoriteEventsPresenterTests/FavoriteEventsPresenterTests.swift

```diff
@@ -106,11 +111,12 @@ class FavoriteEventsPresenterTests: XCTestCase {
                            similarEventsCount: 2,
                            source: EventSource(id: 1, name: "Timepad"),
                            url: URL(string: "https://pitercss.timepad.ru/event/457262/"))
-        eventStorageMock.events = Array(repeating: event1, count: 5)
-        
-        //When
+        let allEvents = Array(repeating: event1, count: 5)
```

Comment:
instead of repeating the same event better to do the following as now any item in your array is the same item:
```swift
let allEvents = createTestEvents()
let firstEvent = allEvents[0]
...
presenter.selectEvent(with: firstEvent.id)
...
XCTAssertEqual(selectedEventServiceMock.selectedEvent, firstEvent)
```

and lets make ```createTestEvents()``` method to create events with unique ids (incrementing is ok) now it is just 1 and 2 repeated.
Why this is bad? Remember you have implemented equatable protocol, now the following code won't throw errors:
```swift
let allEvents = createTestEvents()
XCTAssertEqual(allEvents[0], allEvents[2])
```

Same for SearchEventsPresenterTests.swift